src/client/components/VoyageDetailsContainer.js

```diff
@@ -1,5 +1,180 @@
+            });
+          });
+        axios
+          .get(
+            `/api/suggested-routes/${reportsArr[0].selected_route_id}`,
+            headerObject
+          )
+          .then(res => (res = res.data))
+          .then(selectedRoute => {
+            this.setState({
+              selected_route_table: selectedRoute
+            });
+          });
+      });
+
+    axios
+      .get("/api/voyages/", headerObject)
+      .then(resp => (resp = resp.data))
+      .then(voyageArr =>
+        voyageArr.filter(voyage => voyage.id === parseInt(this.state.voyageId))
+      )
+      .then(voyageobj => {
+        console.log("voyajeobj", voyageobj);
+        axios
+          .get(`/api/vessels/${voyageobj[0].vessel_id}/`, headerObject)
+          .then(res => (res = res.data))
+          .then(vesselobj => {
+            this.setState({
+              vesselName: vesselobj[0].name,
+              departure_time: voyageobj[0].departure_time,
+              hfocosts: voyageobj[0].hfocost,
+              lsfocosts: voyageobj[0].lsfocost,
+              vessel_id: voyageobj[0].vessel_id,
+              depart_from_port: voyageobj[0].depart_from_port,
+              arrive_at_port: voyageobj[0].arrive_at_port
+            });
+          });
+        axios
+          .get(`/api/ports/${voyageobj[0].depart_from_port}/`, headerObject)
+          .then(res => (res = res.data))
+          .then(portobj =>
+            this.setState({
+              departsFrom: portobj[0].name
+            })
+          );
+
+        axios
+          .get(`/api/ports/${voyageobj[0].arrive_at_port}/`, headerObject)
+          .then(res => (res = res.data))
+          .then(portobj =>
+            this.setState({
+              arrivesAt: portobj[0].name
+            })
+          );
+      });
+  }
+
+  render() {
+    console.log(this.state.selected_route_table[0].waypoints);
+    const {
+      vesselName,
+      departsFrom,
+      arrivesAt,
+      departure_time,
+      vessel_eta,
+      date,
+      hsfo,
+      lsfo,
+      hfocosts,
+      lsfocosts,
+      hfo_comsumption,
+      lsfo_comsumption,
+      totalcost,
+      selected_route_table
+    } = this.state;
+    let latitude = parseInt(this.state.latitude);
+    let longitude = parseInt(this.state.longitude);
+    let suggested_route_table = selected_route_table.waypoints;
+
+    return (
+      <>
+        <VoyageDetails
+          vesselName={vesselName}
+          departureFrom={departsFrom}
+          arrivesAt={arrivesAt}
+          etd={departure_time}
+          eta={vessel_eta}
+          date={date}
+          latitude={latitude}
+          longitude={longitude}
+          hsfo={hsfo}
+          ulsfo={lsfo}
+        />
+        <CostWidget
+          totalCost={totalcost}
+          hfoCost={hfocosts}
+          lsfoCost={lsfocosts}
+          hfoConsumption={hfo_comsumption}
+          lsfoConsumption={lsfo_comsumption}
+        />
+      </>
```

Comment:
Once you put the map in there, we will have big problems because I renamed all of the props on the map and on the vessel list components to be camelCased. But if you want to avoid refactoring your stuff you could pass the props through the method I'm using to camelCase all of the keys before sending the props into Map.

You can use the package "camelcase-keys" to do so. See how I'm doing it here: https://github.com/HackYourFuture-CPH/ov-class09-fp/compare/vessel-route-on-map-zaki#diff-2106df1bbc7f38d27d9063944bf1a0bdR26 (remember that you don't have it in your npm dependencies yet)